PYME/LMVis/Extras/animation.py

```diff
@@ -253,7 +253,7 @@ def refill(self):
         self.clear_view()
         for snapshot in self.snapshots:
             index = len(self.snapshots)
-            self.view_table.InsertStringItem(index, snapshot.view_id)
+            self.view_table.InsertItem(index, snapshot.view_id)

```

Comment:
This breaks wx3 compatibility (which we are maintaining until at least Jan 2021)